libs/design/docs/theming.md

```diff
@@ -0,0 +1,36 @@
+# Theming
+
+The **daff-theme** utility library is for theming the daffodil
+design library. It contains many useful mixins, functions, 
+and variables.
```

Comment:
This sounds awkward to me. What do you think about "The **daff-theme** utility library consists of mixins, functions, and variables for theming the daffodil design library."?